### ISSUE TITLE
Is "Run Time" correct?

### ISSUE THREAD

**Issue by lerno** [nordlow/compiler-benchmark#22]
From what I can see "Run Time" is merely invoking the compiler. It doesn't run the `main` created and so consequently the "Run Time" goes down with number of functions. A simple test with C bears this out. I get 28650, 2677 and 269 respectively for 100/10 100/100 100/1000.

Is this intentional?

**Comment by nordlow** [comment]
"Run Time" means time it takes to run the main file that is compiled.

**Comment by lerno** [comment]
Yes, but it doesn't look like that's what it does. More like it's running the compiler without any arguments.

**Comment by nordlow** [comment]
> Yes, but it doesn't look like that's what it does. More like it's running the compiler without any arguments.

That's only for languages that doesn't have separate compilation stage in languages interpreter or just-in-time compiled languages such as Julia.

**Comment by lerno** [comment]
For C and C3 I can only see that "run" runs the compiler.

**Comment by nordlow** [comment]
Measurement of "Run Time" for C3 now behaves correctly on master.
